src/provider/model/ClientHandler.ts

```diff
@@ -0,0 +1,93 @@
+import {Identity} from 'openfin/_v2/main';
+import {ApplicationOption} from 'openfin/_v2/api/application/applicationOption';
+import {injectable, inject} from 'inversify';
+import {Signal} from 'openfin-service-signal';
+
+import {APITopic, Events} from '../../client/internal';
+import {Inject} from '../common/Injectables';
+import {Injector} from '../common/Injector';
+
+import {APIHandler} from './APIHandler';
+import {CollectionMap} from './database/Database';
+
+export type AppInitData = {
+    id: string;
```

Comment:
This would always be an app UUID?

Not sure which convention should take precedence here... That UUID's are called `uuid`, or that database tables are indexed by `id`... :-P

Reply:
the other 2 collections were using id as the index that's why I chose id :) 
Well, in theory `AppInitData` is a database-related type and id field here defines the id of the database entry within in the scope of this structure more than the client's uuid. But I see your point. Can change to uuid if you feel more strongly about that.